CMakeLists.txt

```diff
@@ -202,12 +202,11 @@ endfunction()
 # Parse toolchain generated map file of `target` and display a readable table format.
 #
 function(mbed_generate_map_file target)
-     find_package (Python3)
      add_custom_command(
          TARGET
              ${target}
          POST_BUILD
-         COMMAND ${Python3_EXECUTABLE} ${MBED_PATH}/tools/memap.py -t ${MBED_TOOLCHAIN} ${CMAKE_BINARY_DIR}/${target}${CMAKE_EXECUTABLE_SUFFIX}.map
```

Comment:
can you send this via separate PR (its a bugfix) ? Along with moving find_package to app.cmake